### PR TITLE
[Docs] DiffusionPipeline

### PR DESCRIPTION
**Pull request by patrickvonplaten**

### CODE REVIEW

File: docs/source/api/pipelines/stable_diffusion.mdx [hunk 1]
```diff
@@ -1,39 +1,35 @@
...
 
 Latent diffusion is the research on top of which Stable Diffusion was built. It was proposed in [High-Resolution Image Synthesis with Latent Diffusion Models](https://arxiv.org/abs/2112.10752) by Robin Rombach, Andreas Blattmann, Dominik Lorenz, Patrick Esser, Björn Ommer. You can learn more details about it in the [specific pipeline for latent diffusion](pipelines/latent_diffusion) that is part of 🤗 Diffusers.
 
 For more details about how Stable Diffusion works and how it differs from the base latent diffusion model, please refer to the official [launch announcement post](https://stability.ai/blog/stable-diffusion-announcement) and [this section of our own blog post](https://huggingface.co/blog/stable_diffusion#how-does-stable-diffusion-work).
 
-## Tips
-
+*Tips*:
 - To tweak your prompts on a specific result you liked, you can generate your own latents, as demonstrated in the following notebook: [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/pcuenca/diffusers-examples/blob/main/notebooks/stable-diffusion-seeds.ipynb)
 - TODO: some interesting Tips
```
Comment (pcuenca) [inline docs/source/api/pipelines/stable_diffusion.mdx:11]: lol, I forgot about this, we should remove this line unless someone can think of anything.

Comment (patrickvonplaten) [reply docs/source/api/pipelines/stable_diffusion.mdx:11]: Think it's fine to leave for now :sweat_smile: actually but if you have some ideas feel free to open a PR :-)

**Review by pcuenca — COMMENTED** [review]
Very nice!

File: docs/source/api/pipelines/stable_diffusion.mdx (hunk 1 above)
Comment (pcuenca) [reply docs/source/api/pipelines/stable_diffusion.mdx:11]: haha no I don't :)

File: src/diffusers/pipeline_utils.py [hunk 2]
```diff
@@ -72,6 +72,20 @@ class ImagePipelineOutput(BaseOutput):
 
 
 class DiffusionPipeline(ConfigMixin):
+    r"""
+    Base class for all models.
+
+    [`DiffusionPipeline`] takes care of storing all components (models, schedulers, processors) for diffusion pipelines
+    and handles methods for loading, downloading and saving models as well as a few methods common to all pipelines to:
```
[the remark this answers is not in the document]
Comment (patrickvonplaten) [reply src/diffusers/pipeline_utils.py:79]: Sorry was a bit too fast here in the rush yesterday :sweat_smile: - would you maybe like to open a new PR?

**Pull request merged**